index.html

```diff
@@ -3264,9 +3350,9 @@ <h3>Algorithm</h3>
 
       <ol>
         <li>Initialize <var>result</var> to an empty <a class="changed">map</a>.</li>
-        <li>Initialize <var>default language</var> to <code>@none</code>. If the
-          <var>active context</var> has a <a>default language</a>,
-          set <var>default language</var> to it.</li>
+        <li>Initialize <var>default language</var> to <code>@none</code>.
+          If the <var>active context</var> has a <a>default language</a>,
+          set <var>default language</var> to the <a>language tag</a> from that <a>default language</a>.</li>
```

Comment:
"it" was a bit dry, but this phrasing makes it look like the language tag is somehow an attribute or a part of the default language...
I suggest reverting it to "it"
```suggestion
          set <var>default language</var> to it.</li>
```

Reply:
Maybe we should update other uses like this in the spec, that imply that somehow `language tag` is encoded in `default language`.

Not much of a fan of "it" here, however. Maybe:

> If the _active context_ has a **default language**, set _default language_ to **default language**.